src/components/table/OcTableFiles.vue

```diff
@@ -711,64 +711,210 @@ export default {
       selected: []
     }),
     computed: {
-      resources() {
-        return [
-          {
-            name: "forest.jpg",
-            path: "images/nature/forest.jpg",
-            thumbnail: "https://cdn.pixabay.com/photo/2015/09/09/16/05/forest-931706_960_720.jpg",
-            indicators: [],
-            type: "file",
-            sdate: "Mon, 11 Jan 2021 14:34:04 GMT",
-            owner: [{
+          groupingSettings(){
+      return {
+        groupingAllowed: true,
+        defaultGroupingBy: "None",
+        passedGroupingBy: "",  //None, creation, owner,alphabetically
```

Comment:
Why do we still need this? Shouldn't we use the keys of `groupingFunctions`?

Reply:
- it's no possible to use groupingFunctions (for example first key) for default grouping because the order of enumeration is not garanteed for the objects.
- passedGroupingBy is needed if you want to be able to change which grouping to use from outside of the component

Reply:
Yes, I did not mention `defaultGroupingBy`, only `passedGroupingBy`. Do we really need to declare it here for being able to use it externally? Can't we define it somewhere else internally?
Btw, how do you declare an external grouping dropdown? Can you give me an example to understand?